core/templates/dev/head/i18n.js

```diff
@@ -75,8 +75,9 @@ oppia.config([
   var availableLanguageKeys = [];
   var availableLanguageKeysMap = {};
   for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
-    availableLanguageKeys.push(prop);
-    availableLanguageKeysMap[prop + '*'] = prop;
+    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
```

Comment:
Yeah, it shouldn't be necessary to modify the beforeEach() for 457 tests. Let me take a look. I'll get back to you in a bit!